source/joker_effects.c

```diff
@@ -187,10 +213,10 @@ static JokerEffect half_joker_effect(Joker *joker, Card *scored_card) {
     return effect;
 }
 
-static JokerEffect joker_stencil_effect(Joker *joker, Card *scored_card) {
+static JokerEffect joker_stencil_effect(Joker *joker, Card *scored_card, int scored_when) {
     JokerEffect effect = {0};
-    if (scored_card != NULL)
-        return effect; // if card != null, we are not at the end-phase of scoring yet
+
+    SCORE_INDEPENDANT_ONLY(scored_card, scored_when, effect)
```

Comment:
```suggestion
    SCORE_ON_EVENT_ONLY(JOKER_CALLBACK_INDEPENDANT, scored_when, effect)
```

Batch - Consolidate joker event restriction macros